cmd/release-controller/sync.go

```diff
@@ -527,6 +614,34 @@ func (c *Controller) syncAccepted(release *Release) error {
 	var errs []error
 	newestAccepted := acceptedTags[0]
 	for name, publishType := range release.Config.Publish {
+		_, tagPull, previousTagPull, err, _ := c.getReleaseTagInfo(release.Source.GetName(), newestAccepted.Name, "")
```

Comment:
Nulling the release tag from a method to get the release tag smells bad -- could this be factored better?